src/Features/CSharp/Portable/UseSimpleUsingStatement/UseSimpleUsingStatementCodeFixProvider.cs

```diff
@@ -84,11 +84,25 @@ private static IEnumerable<StatementSyntax> Expand(UsingStatementSyntax usingSta
             var result = new List<StatementSyntax>();
             Expand(result, usingStatement);
 
+            var decendentTrivia = usingStatement.DescendantTrivia().ToList();
+            var comments = new List<SyntaxTrivia>();
+
+            decendentTrivia.ForEach(x =>
+            {
+                if (x.Kind() == SyntaxKind.SingleLineCommentTrivia || x.Kind() == SyntaxKind.MultiLineCommentTrivia)
+                {
+                    comments.Add(x);
+                    comments.Add(SyntaxFactory.EndOfLine(Environment.NewLine));
+                }
+            });
```

Comment:
this seems like it would copy *alllllll* the comments inide the using statement (including trivia associated with internal statements).  this would cause duplications.

Reply:
So, given the bug that we're trying to fix, what we want to do instead is take the leading trivia of the *close curly* of the using statement, and we want to add it as leading trivia to the token that follows the using statement.  that way:

```c#
using (var x = y)
{
   // comment
}

return whatever;
```

Becomes:

```c#
using var x = y;

// comment

return whatever;
```

not:

```c#
// comment
using var x = y;

return whatever;
```